crates/holochain/src/core/workflow/sys_validation_workflow/validation_deps.rs

```diff
@@ -1,40 +1,87 @@
-use holo_hash::ActionHash;
+use holo_hash::HoloHash;
 use holochain_cascade::CascadeSource;
-use holochain_zome_types::{
-    record::{Record, SignedActionHashed},
-    Action,
+use holochain_types::prelude::*;
+use std::{
+    collections::{HashMap, HashSet},
+    sync::Arc,
 };
-use std::collections::{HashMap, HashSet};
+
+/// Cache for tracking dependencies which are fetched during sys validation.
+#[derive(Clone)]
+pub struct ValDeps {
+    /// Dependencies found in the same DHT as the dependent
+    pub local: Arc<parking_lot::Mutex<ValidationDependencies<SignedActionHashed>>>,
```

Comment:
I wonder if `local` is a bit of an overloaded name to use here? e.g. these dependencies aren't necessarily local, they might be remote and need fetching. Maybe `current_dna`?

Reply:
Yes that makes more sense